snippets/MapsUtilsSnippets/Podfile

```diff
@@ -4,6 +4,6 @@ use_frameworks!
 
 target 'MapsUtilsSnippets' do
     platform :ios, '14.0'
-    pod 'GoogleMaps', '8.0.0'
-    pod 'Google-Maps-iOS-Utils', '4.1.0'
+    pod 'GoogleMaps', '7.4.0'
```

Comment:
did you mean to set this to 7.4? Not 9.0?

Reply:
Currently Utils is still only supporting v7.x - the next major releases will support v8 and v9.